src/main/java/com/jasonhhouse/Gaps/GapsApplication.java

```diff
@@ -84,6 +76,109 @@ public void run(String... args) {
         }
         //Always write to command line
         printRecommended();
+
+        if (properties.getMovieDbListId() != null) {
+            createTmdbList();
+        }
+    }
+
+    private void createTmdbList() {
+        // create-request-token
+        OkHttpClient client = new OkHttpClient();
+
+        MediaType mediaType = MediaType.parse("application/octet-stream");
+        RequestBody.create(mediaType, "{}");
+        RequestBody body;
+        Request request = new Request.Builder()
+                .url("https://api.themoviedb.org/3/authentication/token/new?api_key=" + properties.getMovieDbApiKey())
+                .get()
+                .build();
+
+        String request_token;
+        try {
+            Response response = client.newCall(request).execute();
+            JSONObject responseJson = new JSONObject(response.body().string());
+            request_token = responseJson.getString("request_token");
+
+            // Have user click link:
+            logger.info("############################################");
+            logger.info("Click the link below to authorize TMDB list access: " + "https://www.themoviedb.org/authenticate/" + request_token);
+            logger.info("Press enter to continue");
+            logger.info("############################################");
+            System.in.read();
+        } catch (Exception e) {
+            logger.error("Unable to authenticate tmdb, and add movies to list. ");
```

Comment:
You can change this to be a single line.
logger.error("Unable to authenticate tmdb, and add movies to list.", e);